src/interface/interface.json

```diff
@@ -482,10 +482,16 @@
               "desc": "sets the placeholder",
               "params": ["{String} placeholder for our text field"]
             },
+            "getValue": {
+
+            },
             "setValue": {
               "desc": "sets the value",
               "params": ["{String} value for our text field"]
             },
+            "showClearButton": {},
+            "hideClearButton": {},
+            "onClearButtonClick": {},
```

Comment:
I left these blank when we were on the call together, but would you fill them in?

Reply:
Sure!  Definitely WIP here